libcxx/test/support/charconv_test_helpers.h

```diff
@@ -317,6 +317,8 @@ auto all_unsigned = type_list<
     >();
 auto integrals = concat(all_signed, all_unsigned);
 
+auto all_floats = type_list< float, double >(); //TODO: Add long double
```

Comment:
We should refactor this to use `test/support/type_algorithms.h` instead. Doesn't have to be in this patch.